docs/source/develop/api/commands.rst

```diff
@@ -37,6 +38,7 @@ Structure
 
     "Asset ID", "id of the asset", "<asset_name>#<domain_id>", "usd#morgan"
     "Amount", "positive amount of the asset to add", "> 0", "200.02"
+    "Title", "title of the transaction", "title", "title"
```

Comment:
`transaction` may be confusing here, that's applicable to the command, right?

Reply:
@dominious1 any comments?

Reply:
it was corrected in:  https://github.com/hyperledger/iroha/pull/4032/commits/59aecd8908786a699e650dd8b368401cb4222ebb